docs/sources/as-code/observability-as-code/provision-resources/git-sync-setup.md

```diff
@@ -29,62 +29,72 @@ You can sign up to the private preview using the [Git Sync early access form](ht
-1. [Create a GitHub access token](#create-a-github-access-token).
-1. [Configure a connection to your GitHub repository](#set-up-the-connection-to-github).
-1. [Choose what content to sync with Grafana](#choose-what-to-synchronize).
+{{< admonition type="caution" >}}
 
-Optionally, you can [extend Git Sync](#configure-webhooks-and-image-rendering) by enabling pull request notifications and image previews of dashboard changes.
+Refer to [Known limitations](/docs/grafana/latest/observability-as-code/provision-resources/intro-git-sync#known-limitations) before using Git Sync. Refer to [Supported resources](/docs/grafana/latest/observability-as-code/provision-resources/intro-git-sync#supported-resources) for details about which resources you can sync.
 
-| Capability                                            | Benefit                                                                         | Requires                               |
-| ----------------------------------------------------- | ------------------------------------------------------------------------------- | -------------------------------------- |
-| Adds a table summarizing changes to your pull request | Provides a convenient way to save changes back to GitHub.                       | Webhooks configured                    |
-| Add a dashboard preview image to a PR                 | View a snapshot of dashboard changes to a pull request without opening Grafana. | Image renderer and webhooks configured |
+{{< /admonition >}}
 
-{{< admonition type="note" >}}
+The following limitations apply:
 
-Alternatively, you can configure a local file system instead of using GitHub. Refer to [Set up file provisioning](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/observability-as-code/provision-resources/file-path-setup/) for more information.
+- Git Sync only supports dashboards and folders. Alerts, panels, and other resources aren't supported yet.
+- Full instance sync isn't available in Grafana Cloud.
+- In Grafana OSS/Enterprise:
+  - If you try to perform a full instance sync with resources that contain alerts or panels, Git Sync blocks the connection.
+  - You can't create new alerts or library panels after the setup is completed.
+  - If you opted for full instance sync and want to use alerts and library panels, you'll have to delete the synced repository and connect again with folder sync.
 
-{{< /admonition >}}
+## Before you begin
 
-## Performance impacts of enabling Git Sync
+Before you begin, ensure you have the following:
 
-Git Sync is an experimental feature and is under continuous development. Reporting any issues you encounter can help us improve Git Sync.
+- Administration rights in your Grafana organization
+- A GitHub access token. The Grafana UI will prompt you during setup.
+  - If you're using the [Grafana CLI to set up Git Sync](#set-up-git-sync-using-grafana-cli), your token needs the scopes `repo`, `pull_requests`, and `webhooks`
```

Comment:
it doesn't depend of the method. There is a section about the token permissions. Let's link to that :)